x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/components/monitor_details/monitor_alerts/monitor_detail_alerts.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiLoadingSpinner } from '@elastic/eui';
+import React from 'react';
+import { AlertConsumers } from '@kbn/rule-data-utils';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { useParams } from 'react-router-dom';
+import { useRefreshedRangeFromUrl } from '../../../hooks';
+import { SyntheticsDatePicker } from '../../common/date_picker/synthetics_date_picker';
+import { useSelectedLocation } from '../hooks/use_selected_location';
+import { ClientPluginsStart } from '../../../../../plugin';
+
+export const MONITOR_ALERTS_TABLE_ID = 'xpack.observability.slo.sloDetails.alertTable';
+
+export function MonitorDetailsAlerts() {
+  const {
+    triggersActionsUi: { alertsTableConfigurationRegistry, getAlertsStateTable: AlertsStateTable },
+    observability: { observabilityRuleTypeRegistry },
+  } = useKibana<ClientPluginsStart>().services;
+
+  const { monitorId: configId } = useParams<{ monitorId: string }>();
+
+  const selectedLocation = useSelectedLocation();
+  const { from, to } = useRefreshedRangeFromUrl();
+
+  if (!selectedLocation) {
+    return <EuiLoadingSpinner size="xl" />;
+  }
+
+  return (
+    <>
+      <EuiSpacer size="m" />
+      <EuiFlexGroup direction="column" gutterSize="xl">
+        <EuiFlexItem>
+          <SyntheticsDatePicker fullWidth={true} />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <AlertsStateTable
+            alertsTableConfigurationRegistry={alertsTableConfigurationRegistry}
+            configurationId={AlertConsumers.OBSERVABILITY}
+            id={MONITOR_ALERTS_TABLE_ID}
+            data-test-subj="monitorAlertsTable"
+            featureIds={[AlertConsumers.UPTIME, AlertConsumers.OBSERVABILITY]}
```

Comment:
Why do we need to pass `AlertConsumers.OBSERVABILITY` featured here?

Reply:
https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/slo/public/hooks/use_fetch_active_alerts.ts#L58

Reply:
probably not needed, will remove

Reply:
Just to update here, we had a discussion in Slack and this might be needed for Serverless, needs to be verified.